server/src/main/java/org/apache/cassandra/sidecar/coordination/SidecarPeerProvider.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.coordination;
+
+import java.util.Set;
+
+import org.apache.cassandra.sidecar.common.client.SidecarInstance;
+
+/**
+ * Provides a set of Sidecar peers to monitor and failover when they go DOWN
```

Comment:
Since `SidecarPeerProvider` can be used for other tasks other than health monitor, should we have `Provides a set of peers of current Sidecar` in comment?

Reply:
Synced offline. Leaving it as it is as it may be confused with Cassandra specific terminology.